### PR TITLE
Add requirement for documented security processes

### PR DESCRIPTION
**Pull request by lizrice**
As discussed in TOC meeting Feb 16. The idea is not to impose any particular security processes, just to clarify that they must be in place. 
I'm proposing that this should be in place at Incubation level, since we expect it to be in production with some users at that stage.

See also 
* https://github.com/cncf/sig-security/issues/538 where SIG Security have offered to assess whether existing incubated / graduated projects have process in place
* https://github.com/cncf/sig-security/issues/183 on providing a template - TOC doesn't want to *impose* a specific process, but guidance for projects would be helpful

### CODE REVIEW

File: process/graduation_criteria.adoc [hunk 1]
```diff
@@ -29,6 +29,7 @@ To be accepted to incubating stage, a project must meet the sandbox stage requir
  * Demonstrate a substantial ongoing flow of commits and merged contributions.
  * Since these metrics can vary significantly depending on the type, scope and size of a project, the TOC has final judgement over the level of activity that is adequate to meet these criteria
  * A clear versioning scheme.
+ * Clearly documented security processes explaining how to report security issues to the project, and describing how the project provides updated releases or patches to resolve security vulnerabilities
```
Comment (dims) [inline process/graduation_criteria.adoc:32]: s/processes/process/

Comment (lizrice) [reply process/graduation_criteria.adoc:32]: I think these could be more than one process - one for 3rd party to tell them how to report new a security issue, another for documenting how project responds when vulnerabilities are found

Comment (dims) [reply process/graduation_criteria.adoc:32]: sounds good

Comment (dims) [inline process/graduation_criteria.adoc:32]: Could we please showcase some existing / widely used process for a minimum bar here? For example the one from GH itself:

https://docs.github.com/en/code-security/security-advisories/adding-a-security-policy-to-your-repository

Comment (dims) [reply process/graduation_criteria.adoc:32]: when the sig-security folks write up their recommendations, we can switch the url around to what they come up with.

Comment (dims) [reply process/graduation_criteria.adoc:32]: i am fine with this merging as-is

**Comment by caniszczyk** [comment]
let's leave this open until Friday morning, ping @cncf/toc for their +1s

**Comment by dims** [comment]
+1 from m.e

**Comment by lizrice** [comment]
not sure if I need to +1 my own PR but here it is 😀

**Review by saad-ali — COMMENTED** [review]
/lgtm

+1

**Comment by rochaporto** [comment]
+1

**Comment by dzolotusky** [comment]
+1

**Comment by justincormack** [comment]
+1

**Comment by alena1108** [comment]
+1

**Comment by erinaboyd** [comment]
+1

**Pull request merged**